### DISCUSSION TITLE
Markdown list inside blockquote

### DISCUSSION THREAD

**Discussion by D4rkw0lv3s** [requarks/wiki#4732] [Error / Bug Report] [answered]
When you add a list inside a blockquote that is not the default it ignores the tags "{.is-info/error/success/warning}".

Image:
![image](https://user-images.githubusercontent.com/21691729/143508895-89b8f151-92f8-453a-b551-20a44a97a3d4.png)
It should be blue in this case but it ignores the info tag if a list is used....

**Comment by Nyxtorm** [comment]
Hi,

Even today, I have exactly the same problem, the generated HTML applies the style to the `ul` instead of the `blockquote`.

![This is an image](https://i.imgur.com/Dw6IBT4.png)

```markdown
What I'd like to write in markdown editor:

>  Lorem ipsum :
> - Lorem ipsum1
> - Lorem ipsum2
{.is-info}
```

```html
HTML generated by Wiki.js:

<blockquote data-line="0" class="line">
  <p>Lorem ipsum :</p>
  <ul class="is-info">
    <li>Lorem ipsum1</li>
    <li>Lorem ipsum2</li>
  </ul>
</blockquote>

Style I'd like to see:

<blockquote class="is-info">
  <p>Lorem ipsum</p>
  <ul>
    <li>Lorem ipsum1</li>
    <li>Lorem ipsum2</li>
  </ul>
</blockquote>
```

Sorry for my English !

**Comment by NGPixel** [comment, marked as answer]
That's because the parser cannot determine whether the class should be applied to the list or the blockquote. You need to use the [decorate syntax](https://github.com/rstacruz/markdown-it-decorate) instead.

```md
>  Lorem ipsum :
> - Lorem ipsum1
> - Lorem ipsum2
<!-- {blockquote:.is-info} -->
```

**Reply by Nyxtorm** [reply]
Just perfect, thank you ! 👍

**Reply by ag-gaphp** [reply]
Could this information be added to the wiki? Perhaps under the stylings tab for blockquotes or have a little blurb on the Markdown page?

I can't seem to find this information anywhere in the docs, only in this answered/closed discussion topic on Github.

**Reply by NGPixel** [reply]
Added to the docs:
https://docs.requarks.io/en/editors/markdown#decorate-syntax
